src/main/java/seedu/address/ui/EmployeeCard.java

```diff
@@ -51,6 +53,7 @@ public EmployeeCard(Employee employee, int displayedIndex) {
         name.setText(employee.getName().fullName);
         phone.setText(employee.getPhone().value);
         address.setText(employee.getAddress().value);
+        remark.setText(employee.getRemark().value);
```

Comment:
Maybe we could individually test this function but it's probably optional.

Reply:
Yup we can consider that